pkg/controllers/multicluster/serviceexport_controller.go

```diff
@@ -83,7 +65,7 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	// Mark ServiceExport to be deleted, which is indicated by the deletion timestamp being set.
-	isServiceExportMarkedForDelete := serviceExport.GetDeletionTimestamp() != nil
+	isServiceExportMarkedForDelete := !serviceExport.GetDeletionTimestamp().IsZero()
```

Comment:
Will this never be nil?

Reply:
`IsZero()` checks for both nil and zero value.